mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2565,6 +2571,16 @@
     <string name="debug_drawer_tab_tools_tab_creation_tool_title">Tab creation tool</string>
     <!-- The label of the text field in the tab creation tool. -->
     <string name="debug_drawer_tab_tools_tab_creation_tool_text_field_label">Tab quantity to create</string>
+    <!-- The error message of the text field in the tab creation tool when the text field is empty -->
+    <string name="debug_drawer_tab_tools_tab_quantity_empty_error">Text field is empty</string>
+    <!-- The error message of the text field in the tab creation tool when the text field has characters other than digits -->
+    <string name="debug_drawer_tab_tools_tab_quantity_non_digits_error">Please enter positive integers only</string>
+    <!-- The error message of the text field in the tab creation tool when the text field is a zero -->
+    <string name="debug_drawer_tab_tools_tab_quantity_non_zero_error">Please enter a number greater than zero</string>
+    <!-- The error message of the text field in the tab creation tool when the text field is a
+        quantity greater than the max tabs. The first parameter is the maximum number of tabs
+        that can be generated in one operation.-->
+    <string name="debug_drawer_tab_tools_tab_quantity_exceed_max_error">No more than %1$s tabs can be generated in one operation</string>
```

Comment:
@nicholaspoon03 : Hi, pinging you as I see you have landed this string.
Unfortunately, we don't handle proper plural support on mobile (yet) - so the way the string is currently constructed won't work for many languages (for example, Russian - indeed many languages form their plurals differently than English).
One workaround is to change the string to this: `Exceeded the maximum number of tabs (%1$s) that can be generated in one operation` 
More information on plurals [here](mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms) (bottom section covers XML files)

Reply:
Hi @Delphine, thank you for bringing this to my attention. I have created a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1898516) to fix this issue and currently, I am waiting for the patch to be approved. I will keep you updated on when it lands.